### PR TITLE
RDKTV-9523: Enhance getSinkAtmosCapability() API

### PR DESCRIPTION
**Pull request by shashankkumarc**
Reason for change: Added DS HAL implementation for dsGetSinkAtmosCapability. Handled for all audio output ports and host capability
Test Procedure: Refer Ticket
Risks: Low

### CODE REVIEW

File: DisplaySettings/DisplaySettings.json [hunk 1]
```diff
@@ -2513,6 +2524,22 @@
                     "volumeLevel"
                 ]
             }
+        },
+		"AtmosCapabilityChanged": {
+            "summary": "Triggered when the audio sink device Atmos capability is changed.",
+            "params": {
+                "type": "object",
+                "properties": {
+                    "volumeLevel":  {
```
Comment (apatel859) [inline DisplaySettings/DisplaySettings.json:2533]: Fix volumeLevel it is not part of msg?

File: DisplaySettings/DisplaySettings.json [hunk 2]
```diff
@@ -2513,6 +2524,22 @@
...
+                        "summary": "Atmos status",
+                        "type": "boolean",
+                        "example": true
+                    }
+                },
+                "required": [
+                    "Atmos caps statust"
```
Comment (apatel859) [inline DisplaySettings/DisplaySettings.json:2540]: required field should be one of the param

File: DisplaySettings/DisplaySettings.json [hunk 3]
```diff
@@ -951,6 +951,17 @@
         },
         "getSinkAtmosCapability":{
             "summary": "Returns the ATMOS capability of the sink (HDMI0).",
+            "params": {
+                "type":"object",
+                "properties": {
+                    "audioPort": {
+                        "$ref": "#/definitions/audioPort1"
```
Comment (apatel859) [inline DisplaySettings/DisplaySettings.json:958]: could you fix why we have two definition for audioPort1 audioPort0?

File: DisplaySettings/DisplaySettings.json [hunk 4]
```diff
@@ -2513,6 +2524,22 @@
                     "volumeLevel"
```
Comment (apatel859) [inline DisplaySettings/DisplaySettings.json:2524]: why returing valueLEvel for getSinkAtmosCapability

File: DisplaySettings/CHANGELOG.md [hunk 5]
```diff
@@ -15,6 +15,10 @@ All notable changes to this RDK Service will be documented in this file.
 * Changes in CHANGELOG should be updated when commits are added to the main or release branches. There should be one CHANGELOG entry per JIRA Ticket. This is not enforced on sprint branches since there could be multiple changes for the same JIRA ticket during development. 
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
+## [1.1.20] - 2023-05-24
```
[the remark this answers is not in the document]
Comment (shashankkumarc) [reply DisplaySettings/CHANGELOG.md:18]: Yes, I have changed Minor version as this is not a bug fix.

File: DisplaySettings/DisplaySettings.json (hunk 3 above)
Comment (shashankkumarc) [reply DisplaySettings/DisplaySettings.json:963]: Not sure, this is not part of my change.

**Pull request merged**